media/redesign/stylus/newsletter.styl

```diff
@@ -1,17 +1,3 @@
 @require 'includes/mixins';
 @require 'includes/vars';
-
-#apps-newsletter-subscribe {
-    #newsletter-toggle {
-        @extend $checkbox-label-container;
-    }
-
-    .newsletter-setting {
-        display: none; /* Prevents flash of content */
-        padding-left: $grid-spacing;
-
-        .no-js & {
-            display: block;
-        }
-    }
-}
+@require 'components/newsletter';
```

Comment:
Hm. Would it make sense to move the contents of `components/newsletter.styl` here, or do we prefer to keep that in the `components` directory? We keep dashboard styles in a top-level file (`dashboards.styl`) but maybe that's not the goal longer-term.

We do need at least this top-level `newsletter.styl` file because `compile-stylesheets` only sees top-level files. As an alternative, we could remove this file and use `@require 'components/newsletter'` wherever we need the newsletter styles.